client/src/main/java/com/tencent/rss/client/impl/ShuffleWriteClientImpl.java

```diff
@@ -311,6 +313,9 @@ public Roaring64NavigableMap getShuffleResult(String clientType, Set<ShuffleServ
           blockIdBitmap = response.getBlockIdBitmap();
           isSuccessful = true;
           break;
+        } else if (response.getStatusCode() == ResponseStatusCode.INTERRUPTED) {
```

Comment:
Our aim is to change the status of tasks from failed to success?

Reply:
The target of PR is to ignore InterrptException, not mark is as successful

Reply:
Ignore the exception will make the tasks that should be killed, will be successful.

Reply:
InterruptionException may be caused by another reason, not only by kill action.